zebra/zebra_router.c

```diff
@@ -169,6 +172,13 @@ static void zebra_router_free_table(struct zebra_router_table *zrt)
 	XFREE(MTYPE_ZEBRA_NS, zrt);
 }
 
+extern uint32_t zebra_router_get_next_sequence(void)
```

Comment:
does that 'extern' want to be there - thought there was a style thing about that?

Reply:
yep, this was a cut-n-paste error on my part.  I typically write the .h header first and just messed up.